thirdeye/thirdeye-hadoop/src/main/java/org/apache/pinot/thirdeye/hadoop/push/DeleteOverlappingSegmentsInPinot.java

```diff
@@ -79,7 +79,7 @@ private static IdealState computeNewIdealStateAfterDeletingOverlappingSegments(H
       }
     }
     LOG.info("HOURLY segments that can be deleted: {}", hourlySegmentsToDelete.size());
-    LOG.info("Hourly segments to delete {}", hourlySegmentsToDelete.toString().replaceAll(",", "\n"));
+    LOG.info("Hourly segments to delete {}", hourlySegmentsToDelete.toString().replace(",", "\n"));
```

Comment:
This one can be further optimized with passing in char instead of string.

Reply:
Sorry for my late reply. I optimized the string to char instead. Please check. Thanks.